app/frontend/src/features/auth/signup/Signup.test.tsx

```diff
@@ -216,10 +250,51 @@ describe("Signup", () => {
     expect(screen.getByLabelText(USERNAME)).toBeVisible();
   });
 
+  it("displays the guidelines form when guidelines, feedback and email are pending", async () => {
+    const state: SignupFlowRes.AsObject = {
+      needBasic: false,
+      needAccount: false,
+      needAcceptCommunityGuidelines: true,
+      needFeedback: true,
+      needVerifyEmail: true,
+      flowToken: "token",
+    };
+    window.localStorage.setItem("auth.flowState", JSON.stringify(state));
+    render(<View />, {
+      wrapper: getHookWrapperWithClient({
+        initialRouterEntries: [signupRoute],
+      }).wrapper,
+    });
+    expect(
+      await screen.findByText(COMMUNITY_GUIDELINE_TITLES[0])
+    ).toBeVisible();
+  });
+
+  it("displays the guidelines form when only feedback is pending", async () => {
+    const state: SignupFlowRes.AsObject = {
+      needBasic: false,
+      needAccount: false,
+      needAcceptCommunityGuidelines: true,
+      needFeedback: false,
```

Comment:
```suggestion
      needFeedback: true,
```
Otherwise doesn't match test description?